content/doc/developer/plugin-development/dependency-management.adoc

```diff
@@ -38,18 +47,76 @@ and
     +-org.slf4j:jcl-over-slf4j:1.7.26
 ----
 
+This error says that a conflict exists between the versions of `commons-codec` and `jcl-over-slf4j` specified by the plugin versus Jenkins core.
+See link:/doc/developer/plugin-development/updating-parent/#understanding-requireupperbounddeps-failures-and-fixes[Understanding requireUpperBoundDeps failures and fixes] for more information.
```

Comment:
I get a "Page Not Found" when I click on this link.

Reply:
@MarkEWaite This is how the redirected link looks like: https://github.com/jenkins-infra/jenkins.io/blob/ad0f99a50238d6c4675265d8046706e15b899987/doc/developer/plugin-development/updating-parent/#understanding-requireupperbounddeps-failures-and-fixes

I just clicked on the 'Files Changed' section of this PR, then clicked on 'View File', and then clicked on this hyperlink.

Reply:
Oh, should I view these links in the deployed Jenkins site instead?